### PR TITLE
Add up-for-grabs configuration for exiv2

### PR DESCRIPTION
**Pull request by D4N**

### CODE REVIEW

File: _data/projects/exiv2.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- c++
+- c++11
+- image
+- metadata
+- exif
+- iptc
+- xmp
+- cross-platform
+upforgrabs:
+  name: "good first issue"
```
Comment (shiftkey) [inline _data/projects/exiv2.yml]: No need to wrap this in quotes unless there are some special characters

```suggestion
  name: good first issue
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
@D4N one suggestion to tidy this up, otherwise it looks good to me!

File: _data/projects/exiv2.yml (hunk 1 above)
Comment (D4N) [reply _data/projects/exiv2.yml]: Fixed!

**Review by shiftkey — APPROVED** [review]

**Review by shiftkey — APPROVED** [review]
@D4N thanks!

**Pull request merged**